include/stl2/view/transform.hpp

```diff
@@ -41,11 +41,6 @@ STL2_OPEN_NAMESPACE {
 		constexpr transform_view(R base, F fun)
 		: base_(std::move(base)), fun_(std::move(fun)) {}
 
-		template<InputRange O>
-		requires ViewableRange<O> && _ConstructibleFromRange<R, O>
```

Comment:
This change removes all uses of `_ConstructibleFromRange` but doesn't remove `_ConstructibleFromRange`.